health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Response.js

```
@@ -5,58 +5,56 @@ import { ActionBar, SubmitBar, ArrowLeft, ArrowForward } from "@egovernments/dig
+        footerStyles={{}}
+        iconFill=""
+        info=""
+        maxFooterButtonsAllowed={5}
+        message={t(state?.message)}
+        multipleResponses={[]}
+        props={{}}
+        response={t(state?.fileName)}
+        sortFooterButtons
+        style={{}}
+        type="success"
+      ></PanelCard>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)